tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/ElasticSearchViewsSpec.scala

```diff
@@ -84,6 +84,20 @@ class ElasticSearchViewsSpec extends BaseSpec with EitherValuable with CirceEq {
         }
     }
 
+    "fail to create a view with an invalid mapping" in {
+      val invalidMapping =
+        json"""{"mapping": "fail"}"""
+      val payload        = json"""{ "@type": "ElasticSearchView", "mapping": $invalidMapping }"""
+      deltaClient.put[Json](s"/views/$fullId/invalid", payload, ScoobyDoo) { expectBadRequest }
```

Comment:
The `invalid` segment reads as if the view id is invalid, when the issue is the payload. The `payload` could be called `payloadWithInvalidMapping`, and the view id something else than `invalid`. Similar comment on the other test